src/inmanta/moduletool.py

```diff
@@ -1026,9 +1025,14 @@ def install(self, editable: bool = False, path: Optional[str] = None) -> None:
         context of a project.
         """
 
+        # TODO: refine behavior
```

Comment:
Perhaps an option to pass a project path to install in the context of a project?

Reply:
This one I may reserve for a second PR, depending on how much pytest-inmanta is broken